### ISSUE TITLE
In Virtual Cursor mode, JAWS does not read listboxes correctly if they are followed by dashes

### ISSUE THREAD

**Issue by Vladimir-Goncharov-a11y** [FreedomScientific/standards-support#763]
### Summary
1. Rename attached sample file from .txt to .html, open the sample in Chrome.
[odan_397_dashes.txt](https://github.com/FreedomScientific/standards-support/files/12705201/odan_397_dashes.txt)

2. Set PC focus to the first edit field.
3. Switch JAWS to Virtual PC Cursor mode (by pressing Numpad-Plus)
4. Press the down arrow key several times to get the following JAWS transcript. This JAWS  transcript has lines showing when the down key is pressed.

--- JAWS transcript begins ---
> // Down Arrow key pressed 1st time
> Set PC focus to the field above, switch to Virtual PC cursor, press down arrow keys.
> // Down Arrow pressed the 2nd time
> The First Registry
> // Down Arrow pressed 3rd time
>  List box ---
> // Down Arrow pressed 4th time
> the end ---

--- end of JAWS  transcript --- 

Do you see, after 3rd Down Arrow key, JAWS emits "listbox dash dash dash" showing the dashes as an option  of the listbox.
However, these dashes are the first characters of the next paragraph and they are not an option of the listbox.

### Expected result
After 3rd down arrow key pressed, the JAWS transcript should be:
--- Expected JAWS transcript begins ---
> // Down Arrow pressed 3rd time
>   List box
> // Down Arrow pressed 4th time
> --- the end ---

--- end of JAWS  transcript --- 

As you can see in the expected JAWS transcript, "--- The end ---" is visually shown as the whole phrase in one paragraph.

### Actual result
Presently, JAWS  split the last line into 2 lines, and attaches the first part to the listbox in the previous paragraph., showing the rest of the line in the next paragraph.

### Additional Information
Reproducible with JAWS 2023 August release.
Reproducible with JAWS 2024 beta September release
Reproducible with the default JAWS settings (Insert-Space, z)

#### JAWS version and build number
JAWS version 2023.2307.37.400
JAWS version 2024.2308.124.400 

#### Operating System and version
Microsoft Windows 10 Professional Version 22H2 (OS Build 19045.3448), System Type x64 

#### Browser and version:
Google Chrome Version 116.0.5845.188

**Comment by JAWS-test** [comment]
I can confirm the error. It occurs with all punctuation marks, such as hyphen, question mark, exclamation mark, colon, full stop. But not with other special characters (e.g. +, * or #) and not with letters or numbers.

The error does not occur with NVDA.

**Comment by stevefaulkner** [comment]
test file: https://codepen.io/stevef/pen/mdaxyvE

**Comment by BrettLewisVispero** [comment]
Imported into ADO as bug 48219

**Comment by Vladimir-Goncharov-a11y** [comment]
@BrettLewisVispero  this bug is fixed in JAWS 2024 May's update.
Thank you very much!
